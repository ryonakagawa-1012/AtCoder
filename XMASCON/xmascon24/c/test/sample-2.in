4
20 24 12 24
1
10
0 1 1 2 3 5 8 13 21 34
