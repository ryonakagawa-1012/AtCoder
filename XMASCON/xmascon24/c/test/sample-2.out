20 24 36 96 204 456 960 1992 4020 7968
